### PR TITLE
Feature/update packages

### PR DESCRIPTION
**Pull request by doutori**
## Todo
- Update node modules

## Remark

- `next-redux-wrapper` has a few breaking changes, so keeping the version.
  - do it up in later

|before|after|
|----|----|
|before|after|

## Relations

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -51,13 +51,13 @@ app.prepare()
             ctx.body = JSON.stringify({ uptime: process.uptime() });
         });
 
-        router.get('*', async (ctx: ParameterizedContext<Logger>) => {
+        router.get(/.*/, async (ctx: ParameterizedContext<Logger>) => {
```
Comment (doutori) [inline server/index.ts:54]: This implementation in version 8 is gone, so we have to specify it explicitly. 😓 
https://github.com/koajs/router/blob/8.x/lib/route.js#L30

Comment (uruha) [reply server/index.ts:54]: It can not be helped ;) 👍

**Comment by uruha** [comment]
@doutori 
I found removed `snapshot-diff` testing module and jest testing is failed after your commits, so I reinstalled this module ;)

File: server/index.ts (hunk 1 above)
Comment (uruha) [reply server/index.ts:54]: I think that it would be nice if a regular expression could help convey the intent of the filter.

**Comment by doutori** [comment]
Thanks.
When I checked the packages individually, it seems that `snapshot-diff` was deleted.

**Pull request merged**